### PR TITLE
Fixed board styles

### PR DESCRIPTION
**Pull request by DaniilPoiarkov**

### CODE REVIEW

**Review by andpii — DISMISSED** [review]

File: backend/Tasque.BLL/Services/OrganizationService.cs [hunk 1]
```diff
@@ -27,6 +27,7 @@ public override async Task<OrganizationInfoDto> Create(OrganizationCreateDto cre
             entityToCreate.AuthorId = _currentUserId;
 
             await _dbSet.AddAsync(entityToCreate);
+            await _db.SaveChangesAsync();
```
Comment (Its-OP) [inline backend/Tasque.BLL/Services/OrganizationService.cs:30]: do we need to have 2 `.Save` calls in one method ?

File: backend/Tasque.BLL/Services/OrganizationService.cs [hunk 2]
```diff
@@ -36,9 +37,9 @@ public override async Task<OrganizationInfoDto> Create(OrganizationCreateDto cre
             };
 
             await _db.UserOrganizationRoles.AddAsync(orgRole);
-
             await _db.SaveChangesAsync();
 
+
```
Comment (Its-OP) [inline backend/Tasque.BLL/Services/OrganizationService.cs]: remove this line

**Review by andpii — DISMISSED** [review]

File: frontend/src/app/components/tasque-board/tasque-board.component.ts [hunk 3]
```diff
@@ -43,7 +43,8 @@ export class TasqueBoardComponent implements OnInit, OnDestroy {
 
   public unsubscribe$ = new Subject<void>();
 
-  public project: ProjectModel;
+  public project: ProjectModel | undefined;
```
Comment (Its-OP) [inline frontend/src/app/components/tasque-board/tasque-board.component.ts]: can we ever have a board without a project ?

**Review by Its-OP — CHANGES_REQUESTED** [review]

File: backend/Tasque.BLL/Services/OrganizationService.cs (hunk 1 above)
Comment (DaniilPoiarkov) [reply backend/Tasque.BLL/Services/OrganizationService.cs:30]: To get correct organizationId while creating organizationRole for currentUser

**Review by Its-OP — APPROVED** [review]

**Review by andpii — APPROVED** [review]

**Pull request merged**